src/android/IdfaPlugin.java

```diff
@@ -21,7 +21,7 @@ protected void getInfo(CallbackContext callbackContext) throws Exception {
 
         JSONObject result = new JSONObject();
         result.put("aaid", info.getId());
-        result.put("limitAdTracking", info.isLimitAdTrackingEnabled());
+        result.put("isTrackingLimited", info.isLimitAdTrackingEnabled());
```

Comment:
I'd still prefer `trackingLimited` name because it's a field, `isTrackingLimited` looks like a method name.